libcxx/include/chrono

```diff
@@ -2836,7 +3182,7 @@ constexpr hours make24(const hours& __h, bool __is_pm) noexcept
         return __h == hours(12) ? hours(0) : __h;
 }
 
-#endif // _LIBCUDACXX_STD_VER > 17
+#endif // _LIBCUDACXX_STD_VER > 11
```

Comment:
I'm confused by seeing this not many lines after the exact same line, and I'm not seeing an `#if`. Can you match the endifs and ifs in a comment here?

Reply:
You're right, this was messed up. Now that everything is just guarded by `_LIBCUDACXX_STD_VER > 11` I consolidated the ifdef to the minimal number of ifdef blocks.